src/main.ts

```diff
@@ -0,0 +1,15 @@
+import Vue from 'vue';
+import VueResource from 'vue-resource';
+import App from './App.vue';
+import router from './router';
+import store from './store';
+
+Vue.config.productionTip = false;
+
+Vue.use(VueResource);
+
+new Vue({
+  router,
+  store,
+  render: h => h(App)
+}).$mount('#app')
```

Comment:
Is this what binds the app to the html?

Reply:
Yes. It is the default entry file generated by the vue-cli.